otj-server-core/src/main/java/com/opentable/server/PortSelector.java

```diff
@@ -166,6 +166,16 @@ public PortSelection getWithDefault(final String springProperty, final String na
         return portSelection.hasValue() ? portSelection : new PortSelection(springProperty, String.valueOf(defaultV), PortSource.FROM_DEFAULT_VALUE, String.valueOf(defaultV));
     }
 
+    private boolean isOtjActuatorLinked() {
+        try {
+            Class.forName("com.opentable.components.actuate.ActuatorBaseConfiguration");
```

Comment:
its been a while that I have seen one of these in the wild.  
Why do we need to check for the availability of the class? 
(sorry if it's a dumb question)

Reply:
To see if it's in the classPath. This is a pretty fragile way to do it of course.

Reply:
I'd personally use a marker file and comment that it can't be renamed or moved.